### DISCUSSION TITLE
DLRS Org or Org Deployment Instructions

### DISCUSSION THREAD

**Discussion by msteinercrmfirm** [SFDO-Community/declarative-lookup-rollup-summaries#1379] [Q&A]
I'm looking for instructions to properly "deploy" DLRS's created in one sandbox to another.  I've done it before and I know there are "quirks".  

This seems to be the only thing Google finds, but it doesn't seem conclusive.
https://github.com/SFDO-Community/declarative-lookup-rollup-summaries/issues/529

I'm looking for the specific metadata items that need to be deployed and what has to be done manually (activation, etc.)
Apex Classes, Apex Triggers, Dataloading, etc.

**Comment by aheber** [comment]
@msteinercrmfirm I imagine you're past this already but I'll add these here for posterity.

What you're looking for should be the Custom Metadata record for the specific rollup and making sure the Trigger and Test Class generated by DLRS (if your configuration is Realtime or Scheduled) are included.

If you're using Scheduled or Full Scheduled modes you'll need to make sure the target org also has the necessary jobs scheduled as there is no real way to migrate scheduled jobs into a new org.
